thejamesgore/Foxtons-Dashboard
# chore: Remove dependencies causing build errors in production

## Description 🚀
This pull request removes the dependencies that were causing build errors in production, allowing for a successful build and deployment.

## Changes Made 👉
- Removed the problematic dependencies from the `package.json` file
- Updated the `package-lock.json` file to reflect the changes made to the `package.json` file

## Reason for Changes 🔍
The changes were made to ensure that the application can be successfully built and deployed in a production environment, providing a better user experience for users of the application.

## How to Test
To test these changes, you can follow these steps:
- Clone this branch to your local machine
- Install the necessary dependencies with `npm install`
- Attempt to build and deploy the application to a production environment
- Verify that the build and deployment process is successful

## Checklist

## Related Issues
None.

## Other Notes
None.
